test.js

```diff
@@ -27,3 +28,32 @@ test('folder size is correct', t => {
     t.end()
   })
 })
+
+test('sync: folder size is larger than 0', t => {
+  try {
+    const bytes = fastFolderSizeSync('.')
+    t.ok(Number.isFinite(bytes))
+    t.ok(bytes > 0)
+    t.end()
+  } catch (err) {
+    t.error(err)
+  }
```

Comment:
you don' really need this, if an error happens the test will fail. same in the next test